méli-mélo/th-zev/meta.md

```diff
@@ -0,0 +1,32 @@
+---
+feature: thématique
+lang: en
+title: Zero Emission Vehicles (ZEV) Theme
+description: Background colours used for NRCan's ZEV campaign
+componentName: th-zev
+expiry: May 31, 2024
+mainPage: zev.html
+cssClass:
+- bg-zev-purple
+- bg-zev-green
+- btn-zev-purple
+- btn-zev-green
+- panel-zev-green
+- panel-zev-purple
+- zev-header-padding
+a11yStatement: >
+  These colours meet the colour contrast requirements as outlined in WCAG 2.1 AA Success Criterion 1.4.3: Contrast (Minimum).
+  Tested by Eric Goodwin, eric.goodwin at nrcan-rncan.gc.ca. 2023-04-18.
```

Comment:
Considering that both colour are used side by side, this working example trend to show there is a use of color to provide meaning. [SC 1.4.1](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)

Is there a meaning or not? 
* If there is, please update the working example along with the fallback to ensure that information are kept. And add instruction here in the statement or in the peNote to mention that distinction need to be provided through text too.
* If there is no meaning, update this statement to mention it.